spec/factories/lettings_log.rb

```diff
@@ -102,7 +102,7 @@
       layear { 2 }
       waityear { 7 }
       postcode_known { 1 }
-      postcode_full { Faker::Address.postcode }
+      postcode_full { "SW1A 1AA" }
```

Comment:
Is there a reason for needing to fix this value?

Reply:
this is so that a particular URL for API call can be stubbed. This way any logs marked :completed2024 will be fine